src/native/Native.cpp

```diff
@@ -133,9 +243,10 @@ extern "C" {
 
         api().RomOpen(romName);
 
-        config.frameBufferEmulation.enable = 0;
-        config.frameBufferEmulation.aspect = Config::aAdjust;
+        config.frameBufferEmulation.enable = 0;//Frame buffer disabled, will be enabled by OOT
+        //config.frameBufferEmulation.aspect = Config::aAdjust;
         
+        config.generalEmulation.hacks |= hack_subscreen | hack_ZeldaMonochrome;
```

Comment:
Did you verify these hacks weren’t already being applied? I thought these hacks were already applied through the rom name logic?